### PR TITLE
Lowpass filter effect which kicks in after bomb

### PR DESCRIPTION
**Pull request by secretrobotron**
@flukeout @DevonBird I just had to check this out, and it works pretty well.

After the bomb explodes, I apply a lowpass filter to all sound effects (which could be changed so that only select sounds go through the filter). It fades away over time.

### CODE REVIEW

**Comment by flukeout** [comment]
Oh DANG @secretrobotron this is sweeeeeet!

File: battle-pong/src/js/sounds.js [hunk 1]
```diff
@@ -147,9 +189,8 @@ function playSound(name, options){
 
   volume.gain.value = soundOptions.volume; // Should we make this a multiplier of the original?
 
-  panNode.connect(soundContext.destination);
+  panNode.connect(globalBiquadFilter);
   volume.connect(panNode);
   source.connect(volume);
   source.start(0);
-
 }
```
Comment (flukeout) [inline battle-pong/src/js/sounds.js]: Plz add an extra line

**Comment by flukeout** [comment]
Yeah I'm good with this pending the extra line lol. Should run it by @DevonBird as well for tweaking / suggestions. Maybe the effect can last a little less long? 👍

**Comment by DevonBird** [comment]
Just checked out the lowpass effect, and it sounds pretty cool! 

I would have the lowpass effect everything BUT the bomb sound itself. I would also make the effect duration about a third as long with the lowpass starting to open back up very shortly after the bomb explodes. You could also try having the lowpass filter close even farther down than it does currently. 

PS. I added a new version of the Round_Start sound to the folder with no padding at the beginning

Also... I just realized that there are 6 bones for the bones powerup instead of 5. I'll add a 6th Bones_Powerup_Disappear sound to the folder tomorrow.

**Pull request merged**